src/app/docs/confirm/confirm-docs.component.html

```diff
@@ -0,0 +1,11 @@
+<sky-docs-demo-page
+  moduleName="SkyConfirmModule"
+  moduleSourceCodePath="src/app/public/modules/confirm/"
+  pageTitle="Confirm"
+>
+  <sky-docs-demo-page-summary>
+    The confirm service launches simple confirmation dialogs to allow users to confirm actions. The <sky-code>SkyConfirmService</sky-code> provider launches the confirmation dialog. Within the service, you can specify the dialog's message and customize the button text.
+
+  </sky-docs-demo-page-summary>
+
```

Comment:
Should we go ahead and set up the Design tab with the content at https://developer.blackbaud.com/skyux/components/confirm#ux-guidelines? I know the focus is JSDocs comments, so we can always copy that content over later, but it might be worth doing now because we'll need to copy it over now to avoid the possibility that we forget to copy it over before updating the docs.

Reply:
Done! I kept the current format and headings for now until we're ready to work on the updates for these docs.